docs/site/HasMany-relation.md

```diff
@@ -0,0 +1,206 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/HasMany-relation.html
+summary:
+---
+
+## Overview
+
+A `hasMany` relation denotes a one-to-many connection of a model to another
+model through referential integrity. The referential integrity is enforced by a
+foreign key constraint on the target model which usually references a primary
+key on the source model. This relation indicates that each instance of the
+declaring or source model has zero or more instances of the target model. For
+example, in an application with customers and orders, a customer can have many
+orders, as illustrated in the diagram below.
+
+![hasMany relation illustration](./imgs/hasMany-relation-example.png)
+
+The target model, **Order**, has a property, **customerId**, as the foreign key
+to reference the declaring model **Customer's** primary key **id**.
+
+To add a `hasMany` relation to your LoopBack application, you need to perform
+the following steps:
+
+1.  Add a property to your model to access related model instances.
+2.  Modify the source model repository class to provide access to a constrained
+    target model repository.
+3.  Call the constrained target model repository CRUD APIs in your controller
+    methods.
+
+## Defining a hasMany Relation
+
+This section describes how to define a `hasMany` relation at the model level
```

Comment:
I think this sentence is redundant :p

Reply:
In that case, how about something like this: `hasMany relation can be defined by using the @hasMany decorator at the model level.`

I just don't like having description of documentation details in docs. They make me feel like I'm reading abstracts of school reports 😨